tools/ci_testing/examples_test.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Test for example jupyter notebooks."""
+
+import testipynb
```

Comment:
We'll need a script to install this dependency during test time. Could you add this to:
https://github.com/tensorflow/addons/blob/master/tools/ci_build/install/ci_requirements.txt

Reply:
Done - but without specifying a specific version number.
Is that ok?